assets/javascripts/modules/checkout/planDateFilter.jsx

```diff
@@ -16,13 +14,13 @@ define([
                 sixday: (date) => date.day() === 1,
```

Comment:
The default JavaScript locale starts with Sunday as day 0. It's also much more of an American convention than British one. http://momentjs.com/docs/#/get-set/day/